src/main.rs

```diff
@@ -99,64 +129,100 @@ fn main() {
         )
         .arg(
             Arg::new("preserve")
-                .help("Preserve file attributes if possible")
+                .help("Preserve file permissions and timestamps if possible")
                 .short('p')
                 .long("preserve")
                 .action(ArgAction::SetTrue),
         )
         .arg(
             Arg::new("pretend")
-                .help("Do not write any files, only calculate compression gains")
+                .help("Do not write any files, only show compression results")
                 .short('P')
                 .long("pretend")
                 .action(ArgAction::SetTrue),
         )
         .arg(
             Arg::new("strip-safe")
-                .help("Strip safely-removable metadata objects")
+                .help("Strip safely-removable chunks, same as '--strip safe'")
                 .short('s')
                 .action(ArgAction::SetTrue)
                 .conflicts_with("strip"),
         )
         .arg(
             Arg::new("strip")
-                .help("Strip metadata objects ['safe', 'all', or comma-separated list]\nCAUTION: stripping 'all' will convert APNGs to standard PNGs")
+                .help("Strip metadata (safe, all, or comma-separated list)\nCAUTION: 'all' will convert APNGs to standard PNGs")
+                .long_help(format!("\
+Strip metadata chunks, where <mode> is one of:
+
+safe    =>  Strip all non-critical chunks, except for the following:
+                {}
+all     =>  Strip all non-critical chunks
+<list>  =>  Strip chunks in the comma-separated list, e.g. 'bKGD,cHRM'
+
+CAUTION: 'all' will convert APNGs to standard PNGs.
+
+Note that 'bKGD', 'sBIT' and 'hIST' will be forcibly stripped if the color type or bit \
+depth is changed, regardless of any options set.",
+                    StripChunks::KEEP_SAFE
+                        .iter()
+                        .map(|c| String::from_utf8_lossy(c))
+                        .collect::<Vec<_>>()
+                        .join(", ")))
                 .long("strip")
                 .value_name("mode")
                 .conflicts_with("strip-safe"),
         )
         .arg(
             Arg::new("keep")
-                .help("Strip all optional metadata except objects in the comma-separated list")
+                .help("Strip all metadata except in the comma-separated list")
                 .long("keep")
                 .value_name("list")
                 .conflicts_with("strip")
                 .conflicts_with("strip-safe"),
         )
         .arg(
             Arg::new("alpha")
-                .help("Perform additional alpha optimizations")
+                .help("Perform additional alpha channel optimization")
```

Comment:
Down to opinion,but I'd emphasise risk here: 
`"Perform additional, lossy alpha channel optimization"`

Reply:
Hm, I'd rather not label it as "lossy" outright, since it's really "visually lossless". I've clarified this in the long help but can't really make the distinction in the short form.

Reply:
That's great! Thanks.